test/openshift/e2e/ginkgo/parallel/1-120_validate_running_must_gather.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package parallel
+
+import (
+	"context"
+	"fmt"
+	osFixture "github.com/redhat-developer/gitops-operator/test/openshift/e2e/ginkgo/fixture/os"
+	"os"
+	"path"
+	"strings"
+
+	argov1beta1api "github.com/argoproj-labs/argocd-operator/api/v1beta1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/redhat-developer/gitops-operator/test/openshift/e2e/ginkgo/fixture"
+	argocdFixture "github.com/redhat-developer/gitops-operator/test/openshift/e2e/ginkgo/fixture/argocd"
+	fixtureUtils "github.com/redhat-developer/gitops-operator/test/openshift/e2e/ginkgo/fixture/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// MustGatherImage is a good place to insert development image
+// Known to be buggy in 1.17.0
+const MustGatherImage = "registry.redhat.io/openshift-gitops-1/must-gather-rhel8:v1.16.0" // TODO, use tag name with SUT version
```

Comment:
I am unsure what image build/tag to use here (so it does not slip behind and will not require constant updates).

For release testing, this should point to a published image of a must-gather commit that will be released.
For CI, this should point to a published image of a must-gather commit that is tested.

Reply:
@adamsaleh, The image is now configurable. Will point the default to `:latest` once we manage to get it published.